build/ci/community/setup_ci_dependencies.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash -xe
+set +o pipefail
+
+python -m pip install --upgrade pip yq
```

Comment:
please use a specific version, in case they break backward compatibility

Reply:
pip is fine too, but `mikefarah/yq` still has implicit `:latest`.
the point is to avoid external breakage, regardless of which installation tool is used